src/index.js

```diff
@@ -1,2 +1,56 @@
-/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
+/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension, */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function handleClick(plusNum) {
+  plusNum += 1;
+  render(plusNum);
```

Comment:
`Assignment to function parameter 'plusNum'`

파라미터에 재할당을 하면 안됩니다. plusNum에 1을 더해서 결국 렌더 함수에 전달하는게 이 함수의 역할이죠.

아래와 같이 처리하면 어떨까요? 린트 에러가 

```suggestion
function handleClick(plusNum) {
  render(plusNum + 1);
```

Reply:
아........... no-param-reassign 문제에 대한 공식문서를 참고했을 때
이 파라미터를 재사용해서 생긴 문제라고 생각했는데 파라미터에 재할당을 하면 안되는 거였군요!!

Reply:
계산기 문제를 해결하는 데에 영상 강의에서 놓친 부분이 있나해서 영상을 다시 보고 있었습니다 ㅎㅎ 1번 과제 코드부터 수정해볼게요!

Reply:
오! no-param-reassign 문제가 사라졌습니다! 파라미터에 재할당을 하면 안되는군요! 그렇다면 계산기 문제에서도 파라미터에 재할당을 하면 안되겠군요..! 작은 힌트를 얻었네요 ㅎㅎ

Reply:
render랑 handleClick이랑 물려있어서 자리바꿔도 오류가나요ㅠㅠ

Reply:
render 함수 안에 handleClick 함수를 넣었더니 오류가 안나는데 일단 올려볼게요!!